doc/user/content/administration/billing.md

```diff
@@ -0,0 +1,110 @@
+`200cc` is equivalent to 2 compute credits/hour. Larger clusters can process data faster and handle larger data volumes.
+
+{{< tip >}}
+You can resize a cluster at any time, even while the cluster is running, to respond to changes in your workload.
+{{</ tip >}}
+
+Clusters are always "on". However, you can adjust the number of replicas that
+are associated with a cluster. See [Compute cost
+factors](#compute-cost-factors) for more information on compute costs for replicas.
+
+### Compute cost factors
+
+{{< note >}}
+  Materialize provides up-to-date results while also providing
+  strong [consistency guarantees](/get-started/isolation-level/), including
+  strict serializability (the default level). Materialize maintains these
+  up-to-date results via incrementally maintained
+  [indexes](/concepts/indexes/) and incrementally maintained [materialized
+  views](/concepts/views) that perform computation upon insert/update/delete. That is:
+
+  - Materialize performs work on "writes"; i.e., Materialize updates the results
+    when data is ingested.
+
+  - Materialize performs **no** work on "reads" from these objects; i.e.,
+    [`SELECT`s](/sql/select/) from these objects, including ad-hoc queries, are
+    free.
+
+  - Materialize is always "on".
+
+{{</ note >}}
+
+Factors that contribute to compute usage include:
+
+| Factor | Notes       |
+|-------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Each [replica for a cluster](/sql/create-cluster/#replication-factor). | The credit usage for a cluster is measured at a one second granularity for each cluster replica. For a cluster replica, its credit usage begins when it is provisioned and ends when it is deprovisioned. |
+| [Indexes](/concepts/indexes/) and [materialized views](/concepts/views) | As data changes (insert/update/delete), [indexes](/concepts/indexes/) and [materialized views](/concepts/views) perform incremental updates to provide up-to-date results. That is: <br>• Indexes and materialized views perform work on "writes"<br>• Indexes and materialized views perform **no** work on "reads" from these objects; i.e., [`SELECT`s](/sql/select/) and [`SUBSCRIBE`s](/sql/subscribe/) from these objects are **free**. |
+| [Sources](/concepts/sources/) | Depends. |
```

Comment:
Hey @morsapaes -- Jessica recommended that you might have some suggestions on how we want to frame what costs w.r.t. sources.

Reply:
Non-upsert sources consume a negligible amount of resources, both during snapshotting and in steady state. What really costs $$ is sources that have upsert logic (_i.e._ `ENVELOPE UPSERT` or `ENVELOPE DEBEZIUM` Kafka sources). So, the way to phrase this is that it depends on the source type, and the semantics of the source, with upsert sources being cost-intensive because we need to maintain the entire key space in memory for deduplication (well, we can partly spill to disk).